### PR TITLE
removed deprecated name file  from luks-btrfs-subvolumes example

### PR DESCRIPTION
**Pull request by midischwarz12**
I think this is the last name field in the examples.

### CODE REVIEW

File: example/luks-btrfs-subvolumes.nix [hunk 1]
```diff
@@ -9,7 +9,6 @@
           partitions = {
             ESP = {
               label = "EFI";
```
Comment (Mic92) [inline example/luks-btrfs-subvolumes.nix:11]: I think this one as well:

```suggestion
```

**Review by midischwarz12 — COMMENTED** [review]
LGTM

**Comment by cvoges12** [comment]
@Mic92 Anything more that you need?

**Comment by Mic92** [comment]
@mergify queue

**Pull request merged**